README.md

```diff
@@ -41,3 +41,19 @@ _Comparison of two screens where the differences are showed by red rectangles. I
 _The red rectangles outlining missing elements on compared screens. In this case pictures are completely different due to bot protection feature during retesting._
 
 <img src="https://raw.githubusercontent.com/Tesena-smart-testing/WatchUI/master/Img/img_inlog.png" width="850" height="300">
+
+### Development
+
+1. Clone this repository
+2. Create `.venv` `python -m venv .venv`
```

Comment:
this is unnecessarily complicated command, I would say. In future, we should consider Poetry or pipenv as package/dependencies manager